array/sort.mbt

```diff
@@ -177,35 +143,28 @@ fn get_limit(len : Int) -> Int {
   let mut len = len
   let mut limit = 0
   while len > 0 {
-    len = len / 2
+    len = len.land(len - 1)
```

Comment:
Isn't `/ 2` faster? It should be just a right shift, right?

Reply:
I just tested it briefly and it is true that /2 is faster, my fault